coremltools/converters/mil/mil/ops/defs/conv.py

```diff
@@ -15,15 +15,15 @@ class conv(Operation):
     
     Parameters
     ----------
-    x: tensor<[n, C_in, *d_in], T> (Required)
+    x: tensor<[n, C_in, \*d_in], T> (Required)
```

Comment:
`*` or `\*` doesn't seem to cause any difference in the way its rendered. Why change it then?

Reply:
It renders as a hyperlink (an underlined highlighted asterisk) that has no URL, which looks like a mistake (and when clicked, does nothing). These are the `WARNING: Inline emphasis start-string without end-string` errors. I had a very long list of them, which made it difficult to see what the real Sphinx errors are.

Reply:
This took only about an hour to do search/replace throughout all the files.